### PR TITLE
[Ingest Manager] Use new form of fleet API paths

### PR DESCRIPTION
**Pull request by michalpristas**
## What does this PR do?

Aligning paths to be even with https://github.com/elastic/kibana/pull/79193

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: dev-tools/packaging/templates/docker/docker-entrypoint.elastic-agent.tmpl [hunk 1]
```diff
@@ -13,8 +13,8 @@ set -eo pipefail
 # KIBANA_USERNAME - username for accessing kibana API [elastic]
 
 function setup(){
-  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/ingest_manager/setup -H 'kbn-xsrf: true' -u ${KIBANA_USERNAME:-elastic}:${KIBANA_PASSWORD:-changeme}
-  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/ingest_manager/fleet/setup \
+  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/fleet/setup -H 'kbn-xsrf: true' -u ${KIBANA_USERNAME:-elastic}:${KIBANA_PASSWORD:-changeme}
+  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/fleet/setup \
```
Comment (nchaulet) [inline dev-tools/packaging/templates/docker/docker-entrypoint.elastic-agent.tmpl:17]:
```suggestion
 curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/fleet/agents/setup \
```

**Review by nchaulet — APPROVED** [review]
Just the fleet agents setup to fix but otherwise 🚀

**Review by ph — APPROVED** [review]
LGTM

File: dev-tools/packaging/templates/docker/docker-entrypoint.elastic-agent.tmpl [hunk 2]
```diff
@@ -13,8 +13,8 @@ set -eo pipefail
 # KIBANA_USERNAME - username for accessing kibana API [elastic]
 
 function setup(){
-  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/ingest_manager/setup -H 'kbn-xsrf: true' -u ${KIBANA_USERNAME:-elastic}:${KIBANA_PASSWORD:-changeme}
-  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/ingest_manager/fleet/setup \
+  curl -X POST  ${KIBANA_HOST:-http://localhost:5601}/api/fleet/setup -H 'kbn-xsrf: true' -u ${KIBANA_USERNAME:-elastic}:${KIBANA_PASSWORD:-changeme}
```
Comment (michalpristas) [inline dev-tools/packaging/templates/docker/docker-entrypoint.elastic-agent.tmpl:16]: @nchaulet is there any difference in these two calls now? will both exist

**Pull request merged**